modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -117,12 +119,57 @@ You can also create a vector index for relationships with a particular type on a
 ----
 CREATE VECTOR INDEX name IF NOT EXISTS
 FOR ()-[r:REL_TYPE]-() ON (r.embedding)
-OPTIONS {indexConfig: {
+OPTIONS { indexConfig: {
  `vector.dimensions`: $dimension,
  `vector.similarity_function`: $similarityFunction
 }}
 ----
 
+[[configuration-settings]]
+=== Configuration settings
+
+For differences with accepted values between vector index providers, see xref:indexes/semantic-indexes/vector-indexes.adoc#vector-index-providers[].
+
+`vector.dimensions` ::
+The dimensionality of the vectors to be indexed, see xref:indexes/semantic-indexes/vector-indexes.adoc#embeddings[].
+Omitting this setting is possible, and any `LIST<INTEGER | FLOAT>` can be indexed and queried, separated by their dimensionality.
+Setting this value will add additional checks that ensure only vectors with the configured dimensionality will be indexed, and querying the index with a vector of a different dimensionality will return an error. It is recommended to provide the dimensions.
+Accepted values::: `INTEGER` between `1` and `4096` inclusively.
+Default Value::: None, though the setting can be omitted. The setting was mandatory prior to Neo4j 5.23.
+
+`vector.similarity_function` ::
+The name of the similarity function to use to define how similar two vectors are.
```

Comment:
```suggestion
The name of the similarity function to use to assess how similar two vectors are.
```

open for discussion
was thinking about "measure", but after a bit of googling wording, i went with "assess", based on this:

https://kdb.ai/learning-hub/articles/methods-of-similarity/

Reply:
I am unsure if we use define elsewhere. In a way all are correct.

A similarity function assesses similarity, and we measure the similarity between two vectors using a similarity function. Though there are multiple similarity functions, each have different definitions and assess similarity differently, and many are valid (it depends on the question you're asking).
We _define_ what "similar" means for us.

However, I am happy with both alternatives. I think "assess" fits best personally.